src/Components/Projects/ProjectsForm.js

```diff
@@ -1,32 +1,125 @@
-import React, { useState } from 'react';
-import '../FormStyles.css';
-
-const ProjectsForm = () => {
-  const [initialValues, setInitialValues] = useState({
-    title: '',
-    description: ''
-  })
-
-  const handleChange = (e) => {
-    if(e.target.name === 'title'){
-      setInitialValues({...initialValues, title: e.target.value})
-    } if(e.target.name === 'description'){
-      setInitialValues({...initialValues, description: e.target.value})
-    }
-  }
-
-  const handleSubmit = (e) => {
-    e.preventDefault();
-    console.log(initialValues);
-  }
+import React, { useState, useEffect } from 'react';
+import {
+  Flex,
+  Box,
+  Heading,
+  FormControl,
+  FormLabel,
+  Input,
+  Textarea,
+  Button,
+  FormErrorMessage,
+  InputGroup,
+  InputLeftElement,
+  Icon
+} from '@chakra-ui/react';
+import { Formik } from 'formik';
+import { ProjectsSchema } from './ProjectsSchema';
+import { MdTitle, MdImage } from 'react-icons/md';
+
+const ProjectsForm = ({ patchObject }) => {
```

Comment:
Para que esta esta prop de patchObject? es para la edicion? porque no se usa en ningun lado? 
Como haces la edicion?
Si la variable no sirve, eliminarla, pero contestar las otras preguntas

Reply:
La prop patchObject es para la edicion, aun no se usa pero la deje lista para cuando agrega la peticion PATH ya que esas peticiones las estoy creando y las tengo comentadas

Reply:
Pablo, por ahora remuevo la variable. Cuando termine de hacer las peticiones la agrego de nuevo

Reply:
Ok, entonces muevo nuevamente el ticket a in progress hasta que este completo